### PR TITLE
feat(metrics): Add tokio runtime metrics

### PR DESCRIPTION
**Pull request by gurinderu**
## Description
Added configurable tokio metrics into the Prometheus endpoint.

## Motivation
For benchmarks, it is necessary to understand the behavior of our runtime. To simplify, additional runtime metrics from the tokio runtime.

## Additional Notes
 - Added a config for these metrics. By default it is disabled.
 - Fixed the client for tests 

## Checklist
- [x] The code follows the project's coding conventions and style guidelines.
- [x] All tests related to the changes have passed successfully.
- [x] Documentation has been updated to reflect the changes (if applicable).
- [x] All new and existing unit tests have passed.
- [x] I have self-reviewed my code and ensured its quality.
- [x] I have added/updated necessary comments to aid understanding.

## Reviewer Checklist
- [ ] Code has been reviewed for quality and adherence to [guidelines](https://doc.rust-lang.org/1.0.0/style/README.html).
- [ ] Tests have been reviewed and are sufficient to validate the changes.
- [ ] Documentation has been reviewed and is up to date.
- [ ] Any questions or concerns have been addressed.

### CODE REVIEW

File: crates/connected-client/src/connected_client.rs [hunk 1]
```diff
@@ -234,6 +257,7 @@ impl ConnectedClient {
     }
 
     async fn raw_receive(&mut self) -> Result<Particle> {
+        log::info!("raw_receive");
```
Comment (kmd-fl) [inline crates/connected-client/src/connected_client.rs:260]: sure?

File: nox/src/node.rs [hunk 2]
```diff
@@ -572,6 +580,7 @@ mod tests {
 
     #[tokio::test]
     async fn run_node() {
+        log_utils::enable_logs();
```
Comment (kmd-fl) [inline nox/src/node.rs:583]: sure?

File: crates/connected-client/src/connected_client.rs (hunk 1 above)
Comment (gurinderu) [reply crates/connected-client/src/connected_client.rs]: Fixed

File: nox/src/node.rs (hunk 2 above)
Comment (gurinderu) [reply nox/src/node.rs:583]: yep, we will take this logs only on failure

**Pull request merged**